string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,49 @@
+/* exported capitalizeWord */
+
+function capitalizeWord(word) {
+  var capitalizedWord = '';
+  var i;
+
+  var upperCased = word.toUpperCase();
+  var lowerCased = word.toLowerCase();
+
+  if (upperCased === 'JAVASCRIPT') {
+    capitalizedWord += upperCased[0];
+    for (i = 1; i < 4; i++) {
+      capitalizedWord += lowerCased[i];
+    }
+    capitalizedWord += upperCased[4];
+    for (i = 5; i < word.length; i++) {
+      capitalizedWord += lowerCased[i];
+    }
+    return capitalizedWord;
```

Comment:
Nicely done here!! Since this is a special case though, you already know the value you want to return. You can simply `return "JavaScript"`